#
# Pulse valves connected to the LabJack
#
#Header  header          # header
float64[10]  valve_p   
---
float64[10]  valve_p   